components/x-live-blog-wrapper/readme.md

```diff
@@ -81,64 +76,43 @@ import { hydrate } from '@financial-times/x-interaction';
 hydrate();
 ```
 
-### Live updates
-This component exports a function named `listenToLiveBlogEvents` which is used for listening to client side live blog updates. These updates come in the form of server sent events sent by `next-live-event-api`.
-
-This function is used in slightly different ways when rendering the component at the client side vs rendering it at the server side.
-
-#### Client side rendering
-A reference to the actions object should be passed as an argument when calling this function for a client side rendered component.
-```jsx
-import { LiveBlogWrapper, listenToLiveBlogEvents } from '@financial-times/x-live-blog-wrapper';
-
-const actionsRef = actions => {
-    listenToLiveBlogEvents({
-          liveBlogWrapperElementId: 'live-blog-wrapper',
-          liveBlogPackageUuid: 'package-uuid',
-          actions // for client side rendered component only
-    });
-};
+### Inserting posts on the client side
 
-<LiveBlogWrapper articleUrl="https://www.ft.com/content/live_blog_package_uuid"
-    showShareButtons={true}
-    id="live-blog-wrapper"
-    posts={posts}
-    actionsRef={actionsRef}
-    />
-```
-
-#### Server side rendering
-This function should be called after hydrating the component if it is rendered at the server side.
-
-Server side:
-```jsx
-import { Serialiser, HydrationData } from '@financial-times/x-interaction';
-import { LiveBlogWrapper } from '@financial-times/x-live-blog-wrapper';
-
-const serialiser = new Serialiser();
-
-<LiveBlogWrapper articleUrl="https://www.ft.com/content/live_blog_package_uuid"
-    showShareButtons={true}
-    id="live-blog-wrapper"
-    posts={posts}
-    serialiser={serialiser} />
-<HydrationData serialiser={serialiser} />
-```
+When live updates come in you can insert a new post by dispatching an action to the component's wrapper.
 
 Client side:
 ```js
 import { hydrate } from '@financial-times/x-interaction';
-import { listenToLiveBlogEvents } from '@financial-times/x-live-blog-wrapper';
 
 hydrate();
-listenToLiveBlogEvents({
-      liveBlogWrapperElementId: 'live-blog-wrapper',
-      liveBlogPackageUuid: 'package-uuid'
-});
+
+const wrapperElement = document.querySelector(
+    `[data-live-blog-wrapper-id="x-dash-element-id"]`
+);
+
+const post = {
+    id: '00000000-0000-0000-0000-000000000000',
+    ...
+};
+
+const action = 'insert-post';
+// wrapperElement must be the last argument.
```

Comment:
If we made the `id` prop required we might be able to avoid asking people to pass the wrapper which is a bit weird...

Reply:
I don't feel too strongly about this - I do agree with you it might be nicer but I have also approved this for now as I think we could do this part later if we feel strongly about improving it?